packages/lib/src/slider/Slider.tsx

```diff
@@ -248,6 +248,7 @@ const DxcSlider = forwardRef<RefType, SliderPropsType>(
       labelFormatCallback,
       margin,
       size = "fillParent",
+      ariaLabel = "Slider input",
```

Comment:
We need to call it the component, just "Slider". The fact that it is an input is an implementational detail.